b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/application/decodeGetApplication.js

```diff
@@ -17,11 +17,12 @@
  */
 
 import callGetApplication from "../../../apiCall/settings/application/callGetApplication";
+import { commonDecode } from "../../../util/apiUtil/commonDecode";
 
 export default async function decodeGetApplication(session, id) {
 
     try {
-        const res = await callGetApplication(session, id);
+        const res = await commonDecode(() => callGetApplication(session, id), null);
```

Comment:
`return await commonDecode(() => callGetApplication(session, id), null);`

Reply:
Check other places as well

Reply:
Isn't it okay to assign this to variable and return the value ?